index.js

```diff
@@ -2,24 +2,40 @@ const TelegramBot = require("node-telegram-bot-api");
 const AWS = require("aws-sdk");
 const axios = require("axios");
 
-// Initialize AWS SDK
-const sqs = new AWS.SQS();
+// initializing the sqs client
+const sqs = new AWS.SQS({ region: "eu-central-1" });
 
-// Initialize Telegram bot with webhook
-const bot = new TelegramBot('7216921050:AAEISmruLCEXGap4zLcpDyzGyLKWTIBq2SU', {
+// setting up the telegram bot
+const bot = new TelegramBot("7216921050:AAEISmruLCEXGap4zLcpDyzGyLKWTIBq2SU", {
```

Comment:
The key needs to be in a .env file this should never ever be exposed ever on a public repo. We are going to have to delete and recreate before going to production